### DISCUSSION TITLE
Wiederholter Absturz des AddOns HomeKit seit Installation der neuen Version 3.67.10.20230225

### DISCUSSION THREAD

**Discussion by ChrisKasp** [OpenCCU/OpenCCU#2259] [🇩🇪 Fragen&Antworten] [answered]
Seid der Installation der aktuellen Version beendet sich das AddOn HomeKit nach ca. 5 Minuten. Ein Neustart sowie eine Neuinstallation und -konfiguration haben nichts daran geändert.
Bei einem Neustart erhalte ich im addon-uninstall-error.log die Fehlermeldung:

```
start-stop-daemon: warning: killing process 1879: No such process
kill: you need to specify whom to kill
```

Leider komme ich damit nicht weiter.

**Comment by jens-maus** [comment, marked as answer]
> Seid der Installation der aktuellen Version beendet sich das AddOn HomeKit nach ca. 5 Minuten.

Was soll denn bitte das "AddOn HomeKit" genau sein?!?! Mehr Details, mehr Infos!

**Reply by ChrisKasp** [reply]
Vielen Dank für die schnelle Rückmeldung.
Ich bitte um Vergebung. Ich dachte, dass dieses AddOn in der RaspberryMatic Community bekannt sei.
Dieses AddOn stellt ein Interface zu Apple Home - Anwendung dar. Hierüber ist es sehr leicht verschiedene Geräte für die Apple-Welt bereitzustellen. (https://github.com/thkl/hap-homematic/).
Gruß Christian

**Reply by jens-maus** [reply]
Klar ist das Addon bekannt, aber eben nicht als "Addon HomeKit", sondern als "HAP-Homematic".

Auf Grund mehrerer schwerwiegender Probleme mit diesem Addon rate ich aber inzwischen generell von der Nutzunh dieses Addons leider ab, da es leider nur sehr stiefmütterlich gepflegt wird (Letzte Änderung im Projekt im Okt 2022) und es in der Vergangenheit öfters damit Probleme bei Nutzern gegeben hat (wie an deinem Problem zusätzlich zu sehen).

Bitte also an das "HAP-Homematic" Projekt wenden und Verbesserungen anmahnen!

**Reply by ChrisKasp** [reply]
Vielen Dank für Deine Anregung.
